hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -409,12 +408,12 @@ private int getNumOfBuckets(String bucketPrefix)
   }
 
   /**
-   * Parse output into ArrayList with Gson.
+   * Parse output into ArrayList with Jackson.
    * @return ArrayList
    */
-  private ArrayList<LinkedTreeMap<String, String>> parseOutputIntoArrayList()
-      throws UnsupportedEncodingException {
-    return new Gson().fromJson(out.toString(DEFAULT_ENCODING), ArrayList.class);
+  private List<HashMap<String, Object>> parseOutputIntoArrayList() throws IOException {
```

Comment:
Do we need specific HashMap return type ? can we change to Map type without breaking the test cases ?

Reply:
Thanks for the comment @devmadhuu 
I have made the changes.